tools/publisher/src/package.rs

```diff
@@ -90,23 +90,28 @@ impl Package {
 
     /// Returns the expected owners of the crate.
     pub fn expected_owners(&self) -> HashSet<String> {
-        let mut ret = HashSet::new();
-
-        // Crate ownership for SDK crates. Crates.io requires that at least one owner
-        // is an individual rather than a team, so we use the automation user for that.
-        ret.insert(String::from(RUST_SDK_CI_OWNER));
-
-        if self.category.is_sdk() {
-            ret.insert(String::from(RUST_SDK_OWNER));
-        } else if self.handle.name.starts_with("aws-smithy-http-server") {
-            ret.insert(String::from(SMITHY_RS_SERVER_OWNER));
-        } else {
-            ret.insert(String::from(RUST_SDK_OWNER));
-            ret.insert(String::from(SMITHY_RS_SERVER_OWNER));
-        }
+        expected_package_owners(&self.category, &self.handle.name)
+    }
+}
 
-        ret
+/// Returns the expected owners of the crate.
+pub fn expected_package_owners(category: &PackageCategory, package_name: &str) -> HashSet<String> {
+    let mut ret = HashSet::new();
+
+    // Crate ownership for SDK crates. Crates.io requires that at least one owner
+    // is an individual rather than a team, so we use the automation user for that.
+    ret.insert(String::from(RUST_SDK_CI_OWNER));
+
+    if category.is_sdk() {
+        ret.insert(String::from(RUST_SDK_OWNER));
+    } else if package_name.starts_with("aws-smithy-http-server") {
+        ret.insert(String::from(SMITHY_RS_SERVER_OWNER));
```

Comment:
Assuming it's also ok for the python folks? https://github.com/awslabs/smithy-rs/blob/51915bed5f5a3226501bb8c467f1bcf3dbea4276/CODEOWNERS#L11

Reply:
This is the current logic for owner assignment, I haven't made any changes in this PR (apart from tweaking the input parameter types).
We can discuss this after the holidays and adjust if necessary in a separate PR.